keywords/TestServeriOS.py

```diff
@@ -73,12 +91,18 @@ def download(self, version_build=None):
         # Package not downloaded, proceed to download from latest builds
         downloaded_package_zip_name = "{}/{}".format(BINARY_DIR, self.package_name)
         if self.platform == "ios":
-            url = "{}/couchbase-lite-ios/{}/{}/{}".format(LATEST_BUILDS, version, build, self.package_name)
+            if self.build is None:
+                if self.version < "2.0.2":
+                    url = "{}/couchbase-lite/ios/{}/{}".format(RELEASED_BUILDS, self.version, self.package_name)
+                else:
+                    url = "{}/couchbase-lite-ios/{}/{}".format(RELEASED_BUILDS, self.version, self.package_name)
+            else:
+                url = "{}/couchbase-lite-ios/{}/{}/{}".format(LATEST_BUILDS, self.version, self.build, self.package_name)
         else:
-            url = "{}/couchbase-lite-net/{}/{}/{}".format(LATEST_BUILDS, version, build, self.package_name)
+            url = "{}/couchbase-lite-net/{}/{}/{}".format(LATEST_BUILDS, self.version, self.build, self.package_name)
 
         log_info("Downloading {} -> {}/{}".format(url, BINARY_DIR, self.package_name))
-        resp = requests.get(url)
+        resp = requests.get(url, verify=False)
```

Comment:
I hope this is for to go through https url, right?

Reply:
for certificate validation